packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton/BatchDetails.tsx

```diff
@@ -0,0 +1,36 @@
+import { Address } from "viem";
+import { useScaffoldReadContract } from "~~/hooks/scaffold-eth";
+
+type BatchProps = {
+  address: Address;
+};
+
+const zeroAddress = "0x0000000000000000000000000000000000000000";
+
+export const BatchDetails = ({ address }: BatchProps) => {
+  console.log(address);
```

Comment:
It would be nice to remove console.log for production applications. It works for testing, but it's not a good practice.

Reply:
Yeah I should have done that, will adjust it now. Thanks for noticing.